### PR TITLE
examples: update react-router example to use latest libraries

### PR DESCRIPTION
**Pull request by pablopalacios**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Comment by pablopalacios** [comment]
@redonkulus last example is done :)

File: examples/react-router/package.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        "@babel/register": "^7.13.16",
+        "classnames": "^2.3.1",
+        "cookie-parser": "^1.4.5",
+        "csurf": "^1.11.0",
+        "fluxible": "^1.4.0",
+        "fluxible-addons-react": "^1.0.0-beta.1",
+        "fluxible-plugin-fetchr": "^0.3.11",
+        "fluxible-router": "^2.0.0-beta.9.0",
+        "prop-types": "^15.7.2",
+        "react": "^16.14.0",
```
Comment (redonkulus) [inline examples/react-router/package.json:22]: Should we use react 17 for all examples now?

File: examples/react-router/package.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        "react-dom": "^16.14.0",
+        "react-helmet": "^6.1.0",
+        "react-router": "^3.2.6"
```
Comment (redonkulus) [inline examples/react-router/package.json:25]: Is there a newer version of this now?

**Review by redonkulus — COMMENTED** [review]
Looks good. A couple of minor questions.

**Comment by pablopalacios** [comment]
@redonkulus I forgot to mention. There are many breaking changes between `react-router` v3 and v5. I tried to use the latest version of it but meh... It didn't go well. Also, v3 only works with react 16. I can prepare another PR later refactoring it to work with the newest `react-router`, but in this PR I opted to focus only on `fluxible-addons-react`. Would it be fine to updated later?

**Comment by redonkulus** [comment]
Ya that’s totally ok. I was just curious. I know react router has a ton of breaking changes. It’s ok to wait since it’s just an example.

**Pull request merged**